components/error/server.tsx

```diff
@@ -0,0 +1,31 @@
+export const ServerError = ({ message }: { message: string }) => (
```

Comment:
I made the server error it's own thing as it's pretty special and we have to use some very heavy handed techniques to detect that our server streams are dead.